keras_lr_finder/lr_finder.py

```diff
@@ -62,6 +62,29 @@ def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         # Restore the original learning rate
         K.set_value(self.model.optimizer.lr, original_lr)
 
+    def find_generator(self, generator, start_lr, end_lr, batch_size=64, epochs=1):
+        num_batches = epochs * len(generator)
```

Comment:
I think this won't work in all cases. Consider, for example, a case where generator is infinite.
And not all generators will have __len__ defined.